src/main/java/Event.java

```diff
@@ -1,15 +1,15 @@
-public class Event extends Todo{
+public class Event extends Todo {
     protected String startDate;
     protected String endDate;
-    public Event(String description, String startDate, String endDate){
+    public Event(String description, String startDate, String endDate) {
         super(description);
         this.tag = "E";
         this.startDate = startDate;
         this.endDate = endDate;
     }
 
     @Override
```

Comment:
Well done on using '@Override' to indicate the override makes it easier and faster to understand the purpose of the method!

Reply:
Thank you!